emperor/support_files/js/shapes.js

```diff
@@ -0,0 +1,35 @@
+define(['jquery'], function($) {
+
+  var shapes = ['sphere', 'cube', 'cone', 'icosahedron', 'cylinder'];
+
+  var $shapesDropdown = $('<select>');
+  for (var i = 0; i < shapes.length; i++) {
+    $shapesDropdown.append(new Option(shapes[i], shapes[i]));
+  }
+
+  function getGeometry(shape) {
+    var geometry;
+    switch (shape) {
+      case 'sphere':
+        geometry = new THREE.SphereGeometry(0.1, 8, 8);
```

Comment:
Could these all be defined once in the `shapes` object on line 3 if that were an object? Assumes the ability to "copy" a shape. Benefit: the addition of future shapes requires only extending that object defined on line 3